main/src/ca/uwaterloo/flix/api/lsp/CompletionItemLabelDetails.scala

```diff
@@ -0,0 +1,11 @@
+package ca.uwaterloo.flix.api.lsp
+
+import org.json4s.JsonDSL.*
+import org.json4s.*
+
+/**
+  * Represents a `CompletionItemLabelDetails` in LSP.
+  */
+case class CompletionItemLabelDetails(detail: Option[String], description: Option[String]) {
```

Comment:
So CompletionItemLabelDetails is optional and its two fields are also optional? 

Thats a lot of optionality! But if thats how they do it, I suppose we should follow that.